.tekton/scripts/check-partner-tasks.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPTDIR/../.." || exit 1
+
+# Expected partner task directory structure:
+# task/partners/<task name>/<version>/<task file>
+
+if [ ! -e "partners/" ]; then
+    exit 0
```

Comment:
Will it be a good idea to add a message here like `SKIPPED .. since PR is not related to partner task submission`

Reply:
A new commit is added to show a message.